cmd/argocd/commands/logout.go

```diff
@@ -18,6 +18,10 @@ func NewLogoutCommand(globalClientOpts *argocdclient.ClientOptions) *cobra.Comma
 		Use:   "logout CONTEXT",
 		Short: "Log out from Argo CD",
 		Long:  "Log out from Argo CD",
+		Example: `# To log out of argocd
+$ argocd logout
+This can be helpful for security reasons or when you want to switch between different ArgoCD contexts or accounts.
```

Comment:
```suggestion
# This can be helpful for security reasons or when you want to switch between different Argo CD contexts or accounts.
```

Reply:
Thank you for the quick review! Just committed your suggestion